client/tests/integration/smartcontracts/validator_with_admin/src/lib.rs

```diff
@@ -14,7 +14,10 @@ use iroha_validator::{
 extern crate panic_halt;
 
 struct Validator {
-    verdict: Result,
+    transaction_verdict: MaybeUninitialized<TransactionValidationResult>,
```

Comment:
Yes, it's my custom type, because otherwise for transaction it would be `Option<Result<Option<Option<InstructionBox::Output>>>>`. So I decided to add semantic value for a type at least for the first `Option`

Reply:
Which name could you suggest? I agree that it sounds similar, but I have no other ideas

Reply:
> Which name could you suggest?

`VerdictOption`, `MaybeVerdict`

Reply:
I like `MaybeVerdict`, will fix